File: 2021/01/01b.swift
var nums: [Int] = []
while let line = readLine() {
    nums.append(Int(line)!)
}
let sums = zip(zip(nums, nums[1...]), nums[2...]).map {($0.0.0 + $0.0.1 + $0.1)}
print(zip(sums, sums[1...]).filter {$0.0 < $0.1}.count)

File: 2021/01/01b.py
from sys import stdin
nums = list(map(int, stdin.readlines()))
sums = list(sum(triplet) for triplet in zip(nums, nums[1:], nums[2:]))
print(sum(1 for n in zip(sums, sums[1:]) if n[0] < n[1]))
